src/main/java/com/radixdlt/SecurityCritical.java

```diff
@@ -1,14 +1,67 @@
 package com.radixdlt;
 
+import java.lang.annotation.Documented;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Target;
 
 /**
  * Annotation to be used for classes which require additional scrutiny
- * in code review. These classes will be mostly low level classes whose
+ * in code review. These classes will be mostly be low level classes whose
  * correctness is critical for security.
  */
+@Documented
```

Comment:
What does this annotation serve?

Reply:
It ensures the underlying annotation is included in the Javadoc.